packages/fx-core/resource/package.nls.json

```diff
@@ -68,14 +68,14 @@
   "plugins.spfx.questions.framework.title": "Framework",
   "plugins.spfx.questions.webpartName.error.duplicate": "The folder %s already exists. Choose a different name for your component.",
   "plugins.spfx.questions.webpartName.error.notMatch": "%s doesn't match pattern: %s",
-  "plugins.spfx.questions.packageSelect.title": "SharePoint Framework",
+  "plugins.spfx.questions.packageSelect.title": "Install the latest version of SharePoint Framework.",
```

Comment:
The question title here is to let user choose which SharePoint Framework he wants to use, either install the latest one locally or use globally installed version. So I don't think the string works
![image](https://github.com/OfficeDev/TeamsFx/assets/73154171/43a588d1-5e2f-482c-942b-fb0b50b03461)

Reply:
Yes, you are right, I will use another field to distinguish them.